core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -316,17 +324,42 @@ private[spark] class MemoryStore(
     *
     * @param blockId the ID of the block we are freeing space for, if any
     * @param space the size of this block
```

Comment:
This variable name / comment is outdated; `space` is the amount of memory that we'd like to free via eviction, which is not necessarily equal to the whole size of the block.